client/flutter/lib/api/content/schema/advice_content.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:WHOFlutter/api/content/content_bundle.dart';
+import 'package:flutter/cupertino.dart';
+import '../content_loading.dart';
+
+typedef AdviceDataSource = Future<AdviceContent> Function(Locale);
+
+/// Interpret a content bundle as Advice data.
+/// Advice data contains banner text, a recommendation link and text,
+/// and a series of advice items comprising text and image pairs.
+class AdviceContent extends ContentBase {
+  String banner;
+  String recommendations;
+  String recommendationsLink;
+  List<AdviceItem> items;
+
+  static Future<AdviceContent> travelAdvice(Locale locale) async {
+    var bundle = await ContentLoading().load(locale, "travel_advice");
+    return AdviceContent(bundle);
+  }
+
+  AdviceContent(ContentBundle bundle) : super(bundle, schemaName: "advice") {
+    try {
+      this.banner = bundle.getString('banner_html') ?? "";
+      this.recommendations = bundle.getString('recommendations_html') ?? "";
+      this.recommendationsLink =
+          bundle.getString('recommendations_link') ?? "https://www.who.int";
```

Comment:
if no link, should not show link here either

Reply:
#1031